src/content/lib/OpportunityViewContent/OpportunityViewContent.js

```diff
@@ -25,7 +25,7 @@ export class OpportunityViewContent extends React.Component {
   }
 
   handleBack () {
-    Router.push('/opportunities')
+    Router.pushRoute('/job-types')
   }
```

Comment:
why in /opportunity-view back url is /job-types?
we must navigate to /opportunities instead